src/renderer/shared/util/sendAsset.js

```diff
@@ -16,17 +14,38 @@ export default async function sendAsset({ net, asset, amount, receiver, address,
     throw new Error(`Invalid amount: "${amount}"`);
   }
 
-  const selectedAsset = ASSETS[asset];
-  const intents = await api.makeIntent({ [selectedAsset]: amount }, receiver);
-
-  const config = {
-    net,
-    address,
-    privateKey: new wallet.Account(wif).privateKey,
-    intents
-  };
-
-  const { response: { result, txid } } = await api.sendAsset(config, api.neoscan);
+  let result;
+  let txid;
+  if (keys(ASSETS).includes(asset)) {
+    const selectedAsset = ASSETS[asset];
+    const intents = await api.makeIntent({ [selectedAsset]: amount }, receiver);
+
+    const config = {
+      net,
+      address,
+      privateKey: new wallet.Account(wif).privateKey,
+      intents
+    };
+
+    ({ response: { result, txid } } = await api.sendAsset(config, api.neoscan));
+  } else {
+    ({ response: { result, txid } } = await Neon.doInvoke({
+      net,
+      address,
+      script: createScript(asset, 'transfer', [
+        u.reverseHex(wallet.getScriptHashFromAddress(address)),
+        u.reverseHex(wallet.getScriptHashFromAddress(receiver)),
```

Comment:
normally createScript handles this [here](https://github.com/nos/client/blob/5c4d587289ab7036f486291e51c24ef65ca39ec8/src/renderer/browser/util/encodeArgs.js#L15) when encoding is enabled

Although It does not contain the parsing of the Fixed8 float - not sure how we could differentiate it in from a number/Fixed8 using the tree library from @jeroenptrs 

Maybe this is something we should consider?

Reply:
Fixed8 is just a hexstring? Wouldn’t it be very difficult to discern?

Reply:
you could pass in a Fixed8 is literally a float with 8 decimals places :P